build.sbt

```diff
@@ -50,70 +52,50 @@ inThisBuild(
-    )
+    publishArtifact in packageSrc := sys.env.contains("CI")
   )
 )
 
+addCommandAlias("scalafixAll", "all compile:scalafix test:scalafix")
+addCommandAlias("scalafixCheck", "; scalafix --check ; test:scalafix --check")
+
+commands += Command.command("save-expect") { s =>
+  "unit/test:runMain tests.SaveExpect" ::
+    s
+}
+
 lazy val V = new {
   val scala210 = "2.10.7"
   val scala212 = MetalsPlugin.scala212
   val scalameta = MetalsPlugin.semanticdbVersion
-  val scalafix = "0.5.7"
-  val circe = "0.9.0"
-  val enumeratum = "1.5.12"
 }
 
-lazy val legacyScala212 = List(
-  addCompilerPlugin(MetalsPlugin.semanticdbScalac),
-  scalaVersion := V.scala212,
-  crossScalaVersions := List(V.scala212),
-)
+skip.in(publish) := true
 
-skip in publish := true
-legacyScala212
+lazy val mtags = project
+  .settings(
+    libraryDependencies ++= List(
+      "com.thoughtworks.qdox" % "qdox" % "2.0-M9", // for java mtags
+      "org.scalameta" %% "scalameta" % V.scalameta
+    )
+  )
 
 lazy val metals = project
-  .enablePlugins(BuildInfoPlugin)
   .settings(
-    legacyScala212,
-    fork in Test := true, // required for jni interrop with leveldb.
-    buildInfoKeys := Seq[BuildInfoKey](
-      "testWorkspaceBaseDirectory" ->
-        baseDirectory.in(testWorkspace).value,
-      version,
-    ),
-    buildInfoPackage := "scala.meta.metals.internal",
     libraryDependencies ++= List(
+      "com.thoughtworks.qdox" % "qdox" % "2.0-M9", // for java mtags
       "com.lihaoyi" %% "pprint" % "0.5.3", // for pretty formatting of log values
-      "org.scala-sbt.ipcsocket" % "ipcsocket" % "1.0.0", // for sbt server
-      "ch.epfl.scala" % "scalafix-reflect" % V.scalafix cross CrossVersion.full,
-      "com.googlecode.java-diff-utils" % "diffutils" % "1.3.0", // for edit-distance
-      "com.thoughtworks.qdox" % "qdox" % "2.0-M7", // for java mtags
-      "io.get-coursier" %% "coursier" % coursier.util.Properties.version, // for jars
-      "io.get-coursier" %% "coursier-cache" % coursier.util.Properties.version,
-      "io.github.soc" % "directories" % "5", // for cache location
-      "me.xdrop" % "fuzzywuzzy" % "1.1.9", // for workspace/symbol
-      "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8", // for caching classpath index
-      "org.scalameta" %% "lsp4s" % "0.2.1",
-      "org.scalameta" %% "semanticdb-scalac" % V.scalameta cross CrossVersion.full,
-      "io.circe" %% "circe-core" % V.circe,
-      "io.circe" %% "circe-generic" % V.circe,
-      "io.circe" %% "circe-generic-extras" % V.circe,
-      "io.circe" %% "circe-parser" % V.circe,
-      "com.beachape" %% "enumeratum" % V.enumeratum,
-      "com.beachape" %% "enumeratum-circe" % "1.5.15",
-      "org.scalameta" %% "testkit" % V.scalameta % Test,
+      "org.scalameta" %% "scalameta" % V.scalameta,
+      "org.scalameta" %% "symtab" % V.scalameta,
+      "org.scalameta" % "interactive" % V.scalameta cross CrossVersion.full,
```

Comment:
is this still in use?

Reply:
I meant the interactive part specifically, but there's no rush to clean this up now, I agree